.github/workflows/merge.yml

```diff
@@ -804,6 +818,29 @@ jobs:
           SN_LOG: "all"
         timeout-minutes: 10
 
+      - name: Ensure no leftover cash_notes and payment files
+        run: |
+          expected_cash_notes_files="1"
+          expected_payment_files="0"
+          pwd
+          ls $CLIENT_DATA_PATH/ -l
+          ls $CLIENT_DATA_PATH/wallet -l
+          ls $CLIENT_DATA_PATH/wallet/cash_notes -l
+          cash_note_files=$(find $CLIENT_DATA_PATH/wallet/cash_notes -type f | wc -l)
+          if (( $(echo "$cash_note_files > $expected_cash_notes_files" | bc -l) )); then
```

Comment:
Why do we use `bc -l` here instead of using `if [ $expected_cash_notes_files -lt $cash_note_files ]; then` as you have done above?

Reply:
That makes sense. This should be fine. I was just wondering why we were using a different approach there.